libvmi/driver/kvm/kvm.c

```diff
@@ -179,6 +186,51 @@ parse_reg_value(
     }
 }
 
+static reg_t
+parse_seg_reg_value(
+    char *regname,
+    char *ir_output,
+    int type)
+{
+    int offset;
+    int size;
+
+    if (NULL == ir_output || NULL == regname) {
+        return 0;
+    }
+
+    switch(type) {
+        case SEGMENT_SELECTOR:
+        offset = 4;
```

Comment:
These should be indented with another 4 spaces so its easier to see the scope of each case.